Tests/Engine/DataFeeds/LiveTradingDataFeedTests.cs

```diff
@@ -781,38 +781,32 @@ public void FineCoarseFundamentalDataGetsPipedCorrectly()
         [Test]
         public void HandlesCoarseFundamentalData()
         {
-            Symbol symbol = CoarseFundamental.CreateUniverseSymbol(Market.USA);
-            _algorithm.AddUniverse(new FuncUniverse(
-                new SubscriptionDataConfig(typeof(CoarseFundamental), symbol, Resolution.Daily, TimeZones.NewYork, TimeZones.NewYork, false, false, false),
-                new UniverseSettings(Resolution.Second, 1, true, false, TimeSpan.Zero), SecurityInitializer.Null,
-                coarse => coarse.Take(10).Select(x => x.Symbol)
-                ));
+            var symbol = CoarseFundamental.CreateUniverseSymbol(Market.USA);
 
             var lck = new object();
             BaseDataCollection list = null;
-            const int coarseDataPointCount = 100000;
             var timer = new Timer(state =>
             {
-                var currentTime = DateTime.UtcNow.ConvertFromUtc(TimeZones.NewYork);
-                Console.WriteLine(currentTime + ": timer.Elapsed");
-
                 lock (lck)
                 {
-                    list = new BaseDataCollection { Symbol = symbol };
-                    list.Data.AddRange(Enumerable.Range(0, coarseDataPointCount).Select(x => new CoarseFundamental
+                    list = new BaseDataCollection
                     {
-                        Symbol = SymbolCache.GetSymbol(x.ToString()),
-                        Time = currentTime - Time.OneDay, // hard-coded coarse period of one day
+                        Symbol = CoarseFundamental.CreateUniverseSymbol(Market.USA, false)
+                    };
+                    list.Data.AddRange(Enumerable.Range(0, 1000).Select(x => new CoarseFundamental
+                    {
+                        Symbol = Symbol.Create($"{x}", SecurityType.Equity, Market.USA)
                     }));
                 }
-            }, lck, TimeSpan.FromSeconds(3), TimeSpan.FromSeconds(500));
+            }, lck, TimeSpan.FromSeconds(1), Timeout.InfiniteTimeSpan);
```

Comment:
The unit test was actually completly broken 😄 :
- The universe was added before the `DataManager` got hooked to the UniverseManager change event -> so the universe wasn't being added to the data feed at all
- No data was being added to the `BaseDataCollection` since the `SymbolCache.GetSymbol(x.ToString())` threw an exception